MetaMorpheus/EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -83,7 +83,7 @@ private void DoFalseDiscoveryRateAnalysis(FdrAnalysisResults myAnalysisResults)
 
                         // peptides are ordered by MM score from good to bad in order to select the best PSM for each peptide
                         peptides = psms
-                            .OrderByDescending(p => p)
+                            .OrderBy(p => p.FdrInfo.PEP)
                             .GroupBy(p => p.FullSequence)
```

Comment:
should we add a thenby in case two results get the same pep value?

Reply:
You're right, changed it